README-QUIC.md

```diff
@@ -0,0 +1,97 @@
+Using OpenSSL QUIC
+==================
+
+From OpenSSL 3.2, OpenSSL features support for client-side QUIC.
+
+Users interested in using the new QUIC functionality are encouraged to look at
+some of the following resources:
+
+- The [openssl-quic(7) manpage], which provides a basic reference overview of
+  QUIC functionality and how use of QUIC differs from use of TLS with regard to
+  our API;
+- The new [OpenSSL Guide], which provides introductory guides
+  on the use of TLS, QUIC, and other OpenSSL functionality. See the
+  [ossl-guide-introduction(7) manpage] for the index.
+- The [DDD demos], which demonstrate the use of QUIC using simple examples.
+  These can be [found in the source tree under `doc/designs/ddd`].
+- The [demo found in `demos/http3`], which provides an HTTP/3 client example
+  using the nghttp3 HTTP/3 library.
+
+FAQ
+---
+
+### Why would I want to use QUIC, and what functionality does QUIC offer relative to TLS or DTLS?
+
+QUIC is a state-of-the-art secure transport protocol carried over UDP. It can
+serve many of the use cases of TLS as well as those of DTLS. QUIC delivers
+a number of advantages:
+
+- It supports multiple streams of communication, allowing application protocols
+  built on QUIC to create arbitrarily many bytestreams for communication between
+  a client and server. This allows an application protocol to avoid head-of-line
+  blocking and allows an application to open additional logical streams without
+  any round trip penalty, unlike opening an additional TCP connection.
+
+- Since QUIC is the basis of HTTP/3, support for QUIC also enables applications
+  to use HTTP/3 using a suitable third-party library.
+
+- Future implementations of OpenSSL QUIC will offer support for 0-RTT connection
+  initiation, allowing a connection to be initiated to a server and application
+  data to be transmitted without any waiting time. This is similar to TLS 1.3's
+  0-RTT functionality but also avoids the round trip needed to open a TCP
+  socket; thus, it is similar to a combination of TLS 1.3 0-RTT and TCP Fast
+  Open.
+
+- Future implementations of OpenSSL QUIC will offer support for connection
```

Comment:
Same as above `QUIC in OpenSSL`. Otherwise it sounds like OpenSSL QUIC is something different from regular QUIC.